### DISCUSSION TITLE
Getting TLS handshak issue while doing the ubuntu setup.

### DISCUSSION THREAD

**Discussion by swiftsecurity** [mitmproxy/mitmproxy#6918] [Q&A]
I have added the certificates in usr/local/share/ca-certificate/mitmproxy-ca-cert.crt
also run the command sudo update-ca-certificates. Where i can see that 1 added and replacing debian: mitmproxy-ca-cert.pem

But still getting error. I have downloaded the certificate from http://mitm.it official website.

**Comment by Prinzhorn** [comment]
> But still getting error. 

Could you be any more specific? What error? What browser are you using? Not all browsers/tools use the system cert store.
